scalardl-test/build.gradle

```diff
@@ -10,7 +10,8 @@ repositories {
 
 dependencies {
     implementation group: 'com.scalar-labs', name: 'kelpie', version: '1.1.0'
-    implementation group: 'com.scalar-labs', name: 'scalardl-java-client-sdk', version: '2.1.0'
+    implementation group: 'com.scalar-labs', name: 'scalardl-java-client-sdk', version: '3.1.0'
+    implementation group: 'com.google.inject', name: 'guice', version: '5.0.1'
```

Comment:
After using `ClientServiceFactory`, we don't need the dependency on Guice.

Reply:
Sorry, I think we can't remove it because of following source code using this 
https://github.com/scalar-labs/kelpie-test/blob/59f8d77efe2d92045565e2d0f56508a59f69e1d6/scalardl-test/src/main/java/scalardl/transfer/TransferChecker.java#L144